src/puppet-wechat.ts

```diff
@@ -1311,7 +1303,7 @@ export class PuppetWeChat extends PUPPET.Puppet {
       DataLen:       size,
       FileMd5:       fileMd5,
       FromUserName:  fromUserName,
-      MediaType:     WebMediaType.Attachment,
+      MediaType:     4,
```

Comment:
Could you please replace this magic number `4` with the `WebMediaType` enum value?

We should never use any magic number in our code.

Reply:
I made sure that this value is 4 instead of a string by capturing the packet

Reply:
This snippet is the `uploadmediarequest` field  of the `/cgi-bin/mmwebwx-bin/webwxuploadmedia?f=json` request in Web WeChat, we can see that the value of `MediaType` field is `4`.
```json
  {
    "BaseRequest": {
      "Uin": *********,
      "Sid": "***********",
      "Skey": "*********",
      "DeviceID": "********"
    },
    "ClientMediaId": **********,
    "DataLen": 42850,
    "FileMd5": "*********",
    "FromUserName": "**********",
    "MediaType": 4,
    "StartPos": 0,
    "ToUserName": "filehelper",
    "TotalLen": 42850,
    "UploadType": 2
  }
```

Reply:
We can also see the source of the Web WeChat, `MediaType` comes from :   
![image](https://user-images.githubusercontent.com/20336040/144456466-c02093d7-2007-4187-a33f-00dbeed58393.png)
`UPLOAD_MEDIA_TYPE_ATTACHMENT` is defined here:  

![image](https://user-images.githubusercontent.com/20336040/144456822-b7e7be29-bde1-4e9c-be92-8e568606acce.png)

Other media types have never been used.